tests/new_functional/config.js

```diff
@@ -38,6 +39,9 @@ let config = {
   before: function(capabilities, specs) {
     // Allow chaining of browser promises with chai assertions
     chaiAsPromised.transferPromiseness = browser.transferPromiseness;
+
+    browser.addCommand("checkCompletedTick", helpers.checkCompletedTick);
```

Comment:
As discussed, I think we should look at how we can implement this as a helper method in the test rather than as a browser command.